src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+    operate(operation, value) {
+      result = operation(result, value);
```

Comment:
Consider using the arithmetic methods (`add`, `subtract`, `multiply`, `divide`) within the `operate` method to perform operations. This would encapsulate the logic better and make the `operate` method more versatile.